test/route-rate-limit.test.js

```diff
@@ -335,18 +337,20 @@ test('Skip on redis error', async t => {
   t.equal(res.headers['x-ratelimit-limit'], 2)
   t.equal(res.headers['x-ratelimit-remaining'], 1)
 
-  redis.flushall(noop)
-  await redis.quit()
```

Comment:
the scope of this test is to verify the `skipOnError` flag, so without quitting the redis connection we are not testing it